src/api/routes/bns/names.ts

```diff
@@ -63,7 +63,7 @@ export function createBnsNamesRouter(db: DataStore): RouterWithAsync {
         const namePart = name.split('.').slice(1).join('.');
         const resolverResult = await db.getSubdomainResolver({ name: namePart });
         if (resolverResult.found) {
-          res.redirect(resolverResult.result + req.url);
+          res.redirect(`${resolverResult.result}/v1/names${req.url}`);
           next();
```

Comment:
Seems like this should `return` after `next()`?

Reply:
is there a need? idk. I'll add it just to be sure.

Reply:
If you don't then it will fall through to the `res.status(404)` below, and silently error I think. Not a big deal, but better to avoid it